AuRoRA/cosmos.sh

```
@@ -0,0 +1,24 @@
+docker stop cosmos-reason2-2b 2>/dev/null
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)